modules/preview-react/loading-sparkles/stories/examples/Basic.tsx

```diff
@@ -1,6 +1,70 @@
 import React from 'react';
+import {SecondaryButton} from '@workday/canvas-kit-react/button';
+import {Text} from '@workday/canvas-kit-react/text';
 import {LoadingSparkles} from '@workday/canvas-kit-preview-react/loading-sparkles';
+import {AccessibleHide, AriaLiveRegion} from '@workday/canvas-kit-react/common';
+import {createStyles} from '@workday/canvas-kit-styling';
+
+const containerStyles = createStyles({
+  minHeight: '3.5rem',
+  display: 'flex',
+  flexDirection: 'column',
+  gap: '0.5rem',
+});
 
 export const Basic = () => {
-  return <LoadingSparkles aria-live="polite" aria-label="Generating content" />;
+  const [isLoading, setLoading] = React.useState<'idle' | 'loading' | 'success'>('idle');
+  const [quote, setQuote] = React.useState('');
+
+  React.useEffect(() => {
+    if (isLoading === 'loading') {
+      const mockLoading = setTimeout(() => {
+        setLoading('success');
+        setQuote(getQuote());
+      }, 3000);
+
+      return () => {
+        clearTimeout(mockLoading);
+      };
+    }
+  }, [isLoading]);
+
+  const handleClick = () => {
+    setQuote('');
+    setLoading('loading');
+  };
+
+  return (
+    <div>
+      <div>
+        <div className={containerStyles}>
+          {quote && <Text cs={{maxWidth: '60ch'}}>{quote}</Text>}
+          <AriaLiveRegion>
+            {isLoading === 'loading' ? (
+              <LoadingSparkles aria-label="loading" />
+            ) : isLoading === 'success' ? (
+              <AccessibleHide role="status">loading complete</AccessibleHide>
+            ) : (
+              ''
+            )}
```

Comment:
```
            {isLoading === 'loading' && <LoadingSparkles aria-label="loading" />}
            {isLoading === 'success' && (
              <AccessibleHide role="status">loading complete</AccessibleHide>
             }
```

Can we do like this?

Reply:
I thought you liked nested ternaries, @RayRedGoose?!? 😜

Reply:
Just when they look nice, here I don't like empty string 😃